main.go

```diff
@@ -0,0 +1,34 @@
+
+package main
+
+import (
+    "./sreality"
+    _ "./models"
+    "./db"
+    "./config"
+    "./email"
+)
+
+
+
+func main() {
+    config := config.New()
+
+    var urlParameters *sreality.UrlParams = sreality.New(config)
+    adverts := urlParameters.MakeRequest()
+
+    db.Storage = db.NewStorage(config)
```

Comment:
Call it `NewDb()`, basically storages contain business logic, DB stuff and the are intended to be state less